lib/kernel-manager.js

```diff
@@ -50,7 +66,13 @@ class KernelManager {
     });
   }
 
-  startExistingKernel(grammar, connection, connectionFile, onStarted) {
+  // Flow is not aware of Connection, going with Object
```

Comment:
You can export the type [here](https://github.com/nteract/hydrogen/blob/master/lib/zmq-kernel.js#L12) and import it with:
```js
import type { Connection } from "./zmq-kernel.js";
```